tests/std/tests/Dev09_056375_locale_cleanup/__init__.py

```diff
@@ -0,0 +1 @@
+

```

Comment:
Can this contain a license banner and be otherwise empty? Same for the other empty Python files. (This helps if we ever need to add to them in the future.)

Reply:
Will add that.